fares-data-build-tool/fdbt-site
# feat: define time restrictions

# Description

-   This PR introduces a new page which allows the user to define the time restrictions associated with a ticket or set of tickets, providing they have clicked 'Yes' on the previous page. The data collected on this page is added to the matching json of all ticket types.

# Testing instructions

-   Pull down this branch and run the site locally.
-   Navigate through various user journeys on the site and validate that the defineTimeRestrictions page behaves as expected. Also validate that the page format and displays errors correctly.
-   Validate that the matching json produced at the end of each user journey is as expected.

# Type of change


# Checklist:
